### PR TITLE
Document protobuf installation a bit more

### PR DESCRIPTION
**Pull request by srawlins**

### CODE REVIEW

File: tool/grind.dart [hunk 1]
```diff
@@ -312,10 +312,16 @@ void buildbot() => null;
 
 @Task('Generate Protobuf classes')
 void generateProtos() async {
-  await runWithLogging(
-    'protoc',
-    arguments: ['--dart_out=lib/src', 'protos/dart_services.proto'],
-  );
+  try {
```
Comment (domesticmouse) [inline tool/grind.dart:315]: Can you push this `try`/`catch` into the `runWithLogging` function on https://github.com/srawlins/dart-services/blob/docs/tool/grind.dart#L342? 

We can add the informative error message as an optional parameter.

**Comment by domesticmouse** [comment]
Also, you probably want to pull from head. You are trailing `master` by a commit, IIUC.

File: tool/grind.dart (hunk 1 above)
Comment (srawlins) [reply tool/grind.dart]: Done.

**Comment by srawlins** [comment]
Rebased, thanks!

**Review by domesticmouse — APPROVED** [review]
Lgtm!

**Pull request merged**